modules/dreamview/backend/simulation_world/simulation_world_service.cc

```diff
@@ -600,6 +617,18 @@ void SimulationWorldService::SetObstaclePolygon(
   }
 }
 
+void SimulationWorldService::SetObstacleSensorMeasurements(
+    const PerceptionObstacle &obstacle, Object *world_object) {
+  if (world_object == nullptr) {
+    return;
+  }
+  for (const auto &sensor : obstacle.measurements()) {
+    Object *obj = (*(world_.mutable_sensormeasurements()))[sensor.sensor_id()]
```

Comment:
Don't we need clear the old sensor measurement first?

Reply:
clear the old sensor measurement  in Update（） function，like clear_object();